packages/flutter_markdown/test/list_test.dart

```diff
@@ -5,8 +5,6 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_markdown/flutter_markdown.dart';
 import 'package:flutter_test/flutter_test.dart';
-import 'package:markdown/markdown.dart'
```

Comment:
The deprecated versions of these just seem to point to empty strings, so if the tests themselves still pass I think we're good. Otherwise I'll have to dig...